### PR TITLE
Kryoka 426 error isnt displayed on the add lesson page

### PR DESCRIPTION
**Pull request by ferbiy**
## Code reviewers

- [x] @so2niko 
- [x] @CaelumVallis 
- [x] @AllaYakymova 

## Task
https://github.com/ita-social-projects/what-front/issues/426

## Summary of change

1. Added helper class 'CommonHelpers' for dubplicated code in AddLessons.
2. Added function for setting correct error (then we can display error message, instead of true/false values).
3. Changed code style in several places -> es6.
4. Added usage of 'touched' and setFieldTouched (Formik) for displaying errors after click 'Class Register'.
5. Changed markup - moved buttons into form.
6. Added validation for themeName (Lesson Theme) in Yup schema.
7. Changed SRS for field "Lesson Theme" in "Add lesson" page.
8. Divided helpers to cookie and common-helpers.

### CODE REVIEW

**Review by so2niko — APPROVED** [review]
Please, resolve conflicts

File: src/utils/helpers/helpers.js [hunk 1]
```diff
@@ -15,4 +15,10 @@ export class Cookie{
   static del(cname){
       document.cookie = `${cname}=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;`;
   }
+}
+
+export class CommonHelpers {
+  static capitalizeTheme = (str) => str.toLowerCase()
+    .split(/\s+/)
+    .map((word) => word[0].toUpperCase() + word.substring(1)).join(' ');
```
Comment (AllaYakymova) [inline src/utils/helpers/helpers.js:23]: better to move this class in separate file CommonHelpers.js and rename helpers.js into Cookie.js

**Review by AllaYakymova — APPROVED** [review]

File: src/utils/helpers/helpers.js (hunk 1 above)
Comment (ferbiy) [reply src/utils/helpers/helpers.js]: done

**Review by CaelumVallis — APPROVED** [review]
Seems good!

**Pull request merged**